dev/base_include

```diff
@@ -192,7 +192,7 @@ let qid = Libnames.qualid_of_string;;
 (* parsing of terms *)
 
 let parse_constr = Pcoq.parse_string Pcoq.Constr.constr;;
-let parse_tac    = Pcoq.parse_string Ltac_plugin.Pltac.tactic;;
+(*let parse_tac    = Pcoq.parse_string Ltac_plugin.Pltac.tactic;;*)
```

Comment:
oh, I don't know that it should be kept or not. I had a problem whilst debugging where when I used ```#use "include"``` it would give an error on this line. I will revert this.